shm/.github/workflows/main.yml

```diff
@@ -0,0 +1,34 @@
+name: C++ with CMake CI
+
+on: [push, pull_request]
+
+jobs:
+ checker:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@master
+    - name: Run check.sh
+      working-directory: shm
+      run: |
+        chmod +x check.sh
+        sudo apt-get update
+        sudo apt install -y pcregrep
+        ./check.sh CMakeLists.txt
+
+  SHM:
```

Comment:
I am not sure what is wrong with your yaml file, because second job "SHM" should be called even if first job fails. I see that indentaition is different for "checker" and "SHM" but I am not sure if it is a problem. I created '.github/workflows/main.yml' in main project directory, you could try to move this file also.

Reply:
Thank you, I moved github/workflows/main.yml to the main project directory, and for now, I see "This branch has conflicts...". I've checked in my VSC and everything was okay (all built target, tests passed). I don't know if we've done everything right on our side. :)